Wrapping/R/CMakeLists.txt

```diff
@@ -89,10 +102,13 @@ if(NOT SimpleITK_FORBID_DOWNLOADS)
     PRE_BUILD
     ${COPY_DATA_COMMAND}
```

Comment:
Can you add ${EXTRACT_RMAN_COMMAND} here. Then remove the second add_custom_command.

I think that the PRE_BUILD command will be overwritten with the second statement without the APPEND option[1]. Simplifying, to a single add_custom_command should clear that caveat up.
[1] https://cmake.org/cmake/help/v3.0/command/add_custom_command.html